config/config.yaml

```diff
@@ -74,6 +74,11 @@ report:
     # select a sample sheet column for stratification
     by-column: condition
 
+# standardized mutation overview plots with maftools:
+# https://bioconductor.org/packages/release/bioc/vignettes/maftools/inst/doc/maftools.html
+maftools:
+  activate: false
```

Comment:
Do you plan to add all the plots to this PR as well?

Reply:
Yes, exactly. So far, it's the quick fix so everybody can use their own R scripts on the MAF files these initial rules provide. But I'll now also add in rules for the plots. So I'll convert the PR to draft, for now.

Reply:
I think I'll actually opt for reviewing and merging this now, and adding the MAFtools script with plots in a separate pull request.